src/os_auth/local-server.c

```diff
@@ -232,10 +237,23 @@ char* local_dispatch(const char *input) {
                 }
             }
 
+            key_hash = (item = cJSON_GetObjectItem(arguments, "key_hash"), item) ? item->valuestring : NULL;
             key = (item = cJSON_GetObjectItem(arguments, "key"), item) ? item->valuestring : NULL;
-            force = (item = cJSON_GetObjectItem(arguments, "force"), item) ? item->valueint : -1;
 
-            response = local_add(id, name, ip, groups, key, force);
+            if (item = cJSON_GetObjectItem(arguments, "force"), item) {
+                if (item->valueint == -1) {
+                    force_options.enabled = false;
+                }
```

Comment:
According to the style convention, please write `}` and `else` together.

Reply:
Done